test/visibility/submodule-in-file/errors/WrongModule2/SubModule.chpl

```diff
@@ -0,0 +1,5 @@
+module SubModule {
+  writeln("SubModule init");
+  var subVar = 10;
+  proc subFn() { return 20; }
+}
```

Comment:
Maybe I'm just getting tired, but how does this case differ in what it's testing from WrongModule1.chpl?

Reply:
WrongModule1.chpl has the .chpl file name matching the subdirectory name (WrongModule1/).

WrongModule2Wrong.chpl has the module name (WrongModule2) matching the subdirectory name (WrongModule2/).

The SubModule.chpl for both cases are indeed the same.